documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploy Charmed MongoDB
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```shell
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```shell
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Access MongoDB
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. **Do not** directly interface with the admin user in a production environment. In a production environment [always create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead. Later in the section covering Relations we will cover how to access MongoDB without the admin user.*
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
```

Comment:
Why are we using the MongoDB shell?

Reply:
This is MongoDB specific, the MongoDB shell is how most people access MongoDB. Typically the first thing someone does when they deploy MongoDB (charmed or not charmed) is access MongoDB via the shell. :)

Reply:
Yes, I think it does. After all, if the *first* thing they do is access it via the shell, that implies there are more things they can also do, so adding this justification about why we're taking these particular steps is helpful (imo).